### DISCUSSION TITLE
How to disable the italic markdown _ while editing?

### DISCUSSION THREAD

**Discussion by kengjungwa** [outline/outline#1771] [Self Hosting] [answered]
Hi,

While editing, when I type "product_stock_id", the markdown shortcut for italic will be triggered for _stock_; hence the text will be  changed to productstockid (with italic on the word "stock") which is annoying as I have to add _ again and set it to non-italic.

Is there a way to disable italic markdown in the edit mode?

Thanks!

**Comment by tommoor** [comment, marked as answer]
Hi, the easiest might be to use backticks to surround the term with
underscores, this would signify “inline monospaced” and prevent the italic
from triggering

Would that work?

**Reply by kengjungwa** [reply]
Yeah That works! Thanks a lot tommoor for the help and prompt reply 👍
